src/web/jcc/client/components/CommandControl.tsx

```diff
@@ -1272,29 +1274,79 @@ export default class CommandControl extends React.Component {
 				CustomAlert.alert(commDest.poorHealthMessage + commDest.idleStateMessage + commDest.downloadQueueMessage + commDest.disconnectedMessage + notAssignedMessage)
 			} 
 			else {
-
+				const hub = Object.values(this.state.podStatus.hubs)[0]
```

Comment:
Can we create a function for this: getHubStatus. Add some checks to insure there is a hub.